stable/goldilocks/templates/dashboard-deployment.yaml

```diff
@@ -74,11 +74,11 @@ spec:
               protocol: TCP
           livenessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{.Values.dashboard.basePath | default "" }}/health
               port: http
           readinessProbe:
             httpGet:
-              path: {{.Values.dashboard.basePath | default "/" }}health
+              path: {{.Values.dashboard.basePath | default "" }}/health
```

Comment:
I _think_ the way this used to work (prior to the most recent PR that changed it) required a path ending in `/`. So this might be a breaking change.

Reply:
Basepath hasn't existed in Goldilocks for a few versions now, it was recently re-added